src/specializations/_ParametricGeometry.jl

```diff
@@ -0,0 +1,59 @@
+"""
+    _ParametricGeometry <: Meshes.Geometry
+
+_ParametricGeometry is used internally in MeshIntegrals.jl to behave like a generic wrapper
+for geometries with custom parametric functions. This type is used for transforming other
+geometries to enable integration over the standard rectangular `[0,1]^n` domain.
+
+Meshes.jl adopted a ParametrizedCurve type that performs a similar role as of `v0.51.20`,
+but only supports geometries with one parametric dimension. Support is additionally planned
+for more types that span surfaces and volumes, at which time this custom type will probably
+no longer be required.
+
+# Fields
+- `fun::Function` - a parametric function: (ts...) -> Meshes.Point
+"""
+struct _ParametricGeometry{M <: Meshes.Manifold, C <: CRS, F <: Function, Dim} <:
+       Meshes.Primitive{M, C}
+    fun::F
+
+    function _ParametricGeometry{M, C}(
+            fun::F,
+            dims::Int64
```

Comment:
I had this thought too but didn’t think it was worth the implementation effort here since these are only intended for internal use, currently only in six specializations.

Reply:
I agree. I just wanted it to mention here.